Update History of osu! 2015 (#5236)
ppy/osu-wiki: wiki/History_of_osu!/2015/en.md

---
tags:
  - history
  - 2015
  - legacy
---

# History of osu! 2015

## February

Storyboard rules were updated to allow for image sizes larger than 854x640 on 16 February 2015. As  suggested by [Xgor](https://osu.ppy.sh/users/98661), the new maximum rankable image size in storyboards was updated to 1366x768 [[1,2]][r].

## March

The first ever edition of osu!weekly was released on 13 March 2015. Written by [Tasha](https://osu.ppy.sh/users/1031958), osu!weekly positioned itself as "a weekly news digest covering the latest happenings of the game and its community" [[2]][r].

> The first ever osu! Beatmap Blueprint stickers were added to the [osu!store](https://osu.ppy.sh/store/listing) on 20 March 2015. The stickers came in two packs of 8.5" x 11" custom sticker sheets featuring elements such hit circles, combo counters, sliders, and much more [[3]][r]. In celebration of the launch, a contest was run to see who could make the best [beatmaps](/wiki/Beatmap) using the stickers, with a first place prize of an osu! tablet and osu! sticker sets 1 and 2. Despite this, the contest was cancelled due to a lack of submissions, and no follow-up regarding this was ever publicly stated at the time [[3,4]][r]. <!--price needed-->

Author: ```suggestion
The first ever osu! Beatmap Blueprint stickers were added to the [osu!store](https://osu.ppy.sh/store/listing) on 20 March 2015. The stickers came in two packs of 8.5" x 11" custom sticker sheets featuring such elements as hit circles, combo counters, sliders, and much more [[3]][r]. In celebration of the launch, a contest was run to see who could make the best [beatmap](/wiki/Beatmap) using the stickers, with a first place prize of an osu! tablet and osu! sticker sets 1 and 2. Despite this, the contest was cancelled due to a lack of submissions, and no follow-up regarding this was ever publicly stated at the time [[3,4]][r]. <!--price needed-->
```
"despite this" just sounds a bit more natural to me imo; willing to change if you insist

> The first ever osu! Beatmap Blueprint stickers were added to the [osu!store](https://osu.ppy.sh/store/listing) on 20 March 2015. The stickers came in two packs of 8.5" x 11" custom sticker sheets featuring elements such hit circles, combo counters, sliders, and much more [[3]][r]. In celebration of the launch, a contest was run to see who could make the best [beatmaps](/wiki/Beatmap) using the stickers, with a first place prize of an osu! tablet and osu! sticker sets 1 and 2. Despite this, no actual follow-up towards this contest was ever publicly announced, thus it is presumed that the contest was cancelled [[3,4]][r]. <!--official verification needed; price needed-->

Reviewer: There were too few submissions to actually make the contest happen.

[Wii Remote](https://en.wikipedia.org/wiki/Wii_Remote "Wikipedia") support was added for [osu!taiko](/wiki/Game_mode/osu!taiko) on 26 March 2015 [[5]][r]. <!--changelog entry missing-->

> On 31 March 2015, [osu!coins](/wiki/osu!coins) were added as a joke in-game currency for osu! as a part of an April Fools' day joke. The coins were made to parody [free-to-play](https://en.wikipedia.org/wiki/Free-to-play "Wikipedia") mobile games' common use of in-game currencies and [microtransactions](https://en.wikipedia.org/wiki/Microtransaction "Wikipedia"). The osu!coins were quickly removed the following day.

Reviewer 2: Shall we link to https://osu.ppy.sh/wiki/en/History_of_osu%21/April_Fools#2015

Author: Oh, yes, we should. At the time of writing the April Fools' pr was still in review, which is why it wasn't originally linked

Reviewer 2: > Oh, yes, we should. At the time of writing the April Fools' pr was still in review, which is why it wasn't originally linked

Should be https://osu.ppy.sh/wiki/zh/History_of_osu%21/April_Fools#2015 
or https://osu.ppy.sh/wiki/zh/osu%21coin
https://osu.ppy.sh/wiki/zh/History_of_osu%21/April_Fools#osu!coin is invalid

Author: the Chinese version (zh.md) is invalid because there is no Chinese translation for this article. `/wiki/History_of_osu!/April_Fools#osu!coins` should redirect accordingly

## April

The `#balkan` channel was re-added to the game as per a [feature request](https://osu.ppy.sh/community/forums/topics/152009&start=0) on 2 April 2015 [[6,7]][r].

The [Co-op mod](/wiki/Game_modifier/Co-op) became an unranked mod on 6 April 2015 [[8]][r].

9K was made rankable after being given proper use support in the editor on 30 April 2015 [[9]][r].

## May

*Notice: The [forum post](https://osu.ppy.sh/community/forums/topics/334994) that announced news of the new ranking system was released on 4 June 2015, although the old system was actually fully deprecated on 1 May 2015, and full implementation of the new system was still yet to be available to the public by the end of 2015.*

The old Beatmap Nominator (BN) ranking was fully deprecated on 1 May 2015, and replaced with a new, real-time ranking. This new ranking retained the original goal of the previous one, although it focused on challenge and collaboration as opposed to individual performance over a set timeframe. Public reception towards this new change was overall mostly positive, although many users did remark that the new system was confusing [[10,11]][r].

> Among the changes and announcements made surrounding beatmap ranking, [a change was made in the ranking process](https://osu.ppy.sh/community/forums/topics/325973) on 5 May 2015 that allowed players to dispute disqualifications made by the Quality Assurance Team (QAT), due to the lack of any way to dispute issues that the QAT may bring up. The new change made it from then on possible to submit a form if there was a dispute, where a final decision could be made regarding the map and whether it is ready to be ranked [[12]][r].

Reviewer 3: ```suggestion
Among the changes and announcements made surrounding the beatmap ranking, [a change was made to the ranking process](https://osu.ppy.sh/community/forums/topics/325973) on 5 May 2015 that allowed players to dispute disqualifications made by the Quality Assurance Team (QAT), due to the lack of any way to dispute issues that the QAT may bring up. The new change made it possible to submit a form if there was a dispute, where a final decision could be made regarding a beatmap, and whether it is ready to be ranked [[12]][r].
```

Author: ```suggestion
Among the changes and announcements made surrounding beatmap ranking, [a change was made to the ranking process](https://osu.ppy.sh/community/forums/topics/325973) on 5 May 2015 that allowed players to dispute disqualifications made by the Quality Assurance Team (QAT), due to the lack of any way to dispute issues that the QAT may bring up. The new change made it possible to submit a form if there was a dispute, where a final decision could be made regarding a beatmap and whether it is ready to be ranked [[12]][r].

```

> The [osu!news YouTube Channel](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ "YouTube") uploaded its [first video](https://www.youtube.com/watch?v=KQbudVxEjr8 "YouTube") to the public on 8 May 2015. The channel was run by various members of the community and aimed to create videos on skin showcases, tournaments, mapping, contests, and other news from the community [[12,13]][r].

Reviewer 3: ```suggestion
The [osu!news YouTube channel](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ "YouTube") uploaded its [first video](https://www.youtube.com/watch?v=KQbudVxEjr8 "YouTube") to the public on 8 May 2015. The channel was run by various members of the community and aimed to create videos on skin showcases, tournaments, mapping, contests, and other news from the community [[12,13]][r].
```

The osu!keyboard (also known as the "osu! 'nono' keyboard") was added to the osu!store on 30 May 2015, retailing for about $40 at launch. Initially announced in a [tweet by peppy](https://twitter.com/ppy/status/603797988742336512) on 28 May 2015, there were only 100 keyboards available at launch, and were more akin to a keypad than a full-sized keyboard [[14,15]][r].

> ![](img/osu-keyboard.jpg "Image of the osu! keyboard teased in the tweet by peppy")

Reviewer 3: ```suggestion
![](img/osu-keyboard.jpg "The osu! keyboard teased in a tweet by peppy")
```

## June

> Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players must pay $4 USD for their first name change, $8 USD for their second, $16 USD for their third, after which the price would double with each succeeding purchase with a cap at $100 USD; however, if a user had bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].

Reviewer 3: ```suggestion
Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players had to pay $4 USD for their first name change, after which the price would double with each succeeding purchase with a cap at $100 USD. However, if a user bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
```

Author: use of semicolon was intentional
```suggestion
Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players had to pay $4 USD for their first name change, after which the price would double with each succeeding purchase with a cap at $100 USD; however, if a user bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
```

> The ability to see what a replay would look like without [mods](/wiki/Game_modifier) was added on 23 June 2015 as per a [feature request](https://osu.ppy.sh/community/forums/topics/97560), although the change only made the mods [Double Time](/wiki/Game_modifier/Double_Time), [Half Time](/wiki/Game_modifier/Half_Time), and [Flashlight](/wiki/Game_modifier/Flashlight) toggle-able [[20]][r].

Reviewer 3: ```suggestion
The ability to see what a replay would look like without [mods](/wiki/Game_modifier) was added on 23 June 2015 as per a [feature request](https://osu.ppy.sh/community/forums/topics/97560), although the change only made the mods [Double Time](/wiki/Game_modifier/Double_Time), [Half Time](/wiki/Game_modifier/Half_Time), and [Flashlight](/wiki/Game_modifier/Flashlight) togglable [[20]][r].
```

## July

> On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the osu! team directly [[21]][r].

Reviewer 3: ```suggestion
On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the osu! team directly [[21]][r].
```
```suggestion
On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test new cutting-edge features, contribute to graphical/sound resources, or get help working with osu! services could come to join the team and talk to the osu! team directly [[21]][r].
```

Author: the writing here was intentional; "to" and "new" change the meaning

Notwithstanding, the new team was completely public and available for anyone to join, which caused many unwanted users to flood the channels. This forced peppy into "locking down" the new team the very next day. Despite this, peppy continued to make use of the team throughout the rest of the year, and often invited users to come and join when they were testing new features or implementations [[22,23]][r].

## August

The Chat Moderators were officially merged into the Global Moderation Team (GMT) on 20 August 2015 [[24,25]][r].

## September

The old FPS counter was replaced with a new "refresh and latency display" on 2 September 2015. This new display would still display the current FPS (frames per second), but the new change caused it to hide from view if it exceeded 500 FPS. In addition, a new counter was added below the FPS counter that would display exactly how much latency was being introduced during frame rendering (measured in milliseconds). "Stuttered" or "dropped" frames would be shown as coloured rectangles to the left of the counter to better visualise exactly how much frame render latency a user was experiencing [[26,27]][r].

> On 26 September 2015, [cYsmix](http://cysmix.com/ "cysmix.com") became one of the first official osu! artists. This partnership entailed the creation of four tracks made specifically for osu!, along with the creation of a mapping contest using those tracks. The aforementioned four tracks were as follows:

Reviewer 3: > The aforementioned four tracks were as follows:

seems like there are only three tracks below 🧐

Author: woops 
```suggestion
On 26 September 2015, [cYsmix](http://cysmix.com/ "cysmix.com") became one of the first official osu! artists. This partnership entailed the creation of three tracks made specifically for osu!, along with the creation of a mapping contest using those tracks. The aforementioned four tracks were as follows:
```

- ["Manic"](https://soundcloud.com/olemlanglie/cysmix-manic-osu)
- ["House with Legs"](https://soundcloud.com/olemlanglie/cysmix-house-with-legs-osu)
- ["Dovregubben's Hall (New Ver.)"](https://soundcloud.com/olemlanglie/cysmix-dovregubbens-hall-new-ver-osu)

![](img/cysmix-album-cover-1.jpg "The album cover for cYsmix's first release of official osu! songs")

> Each track had its own pre-timed [`.osz`](/wiki/osu!_File_Formats/Osz_(file_format)) file included with it and were released on [cYsmix's SoundCloud](https://soundcloud.com/olemlanglie "SoundCloud") on 28 September 2015.

Reviewer 3: ```suggestion
Each track had its own pre-timed [`.osz`](/wiki/osu!_File_Formats/Osz_(file_format)) file included with it. They were released on [cYsmix's SoundCloud](https://soundcloud.com/olemlanglie "SoundCloud") on 28 September 2015.
```

Author: one sentence sounds a little better imo; will unresolve if you disagree

> In addition to this, an "open bounty" named "Mapping With Rewards" was created for this event, that allowed anyone to map one of the aforementioned tracks, get it into a rankable state, and if it was impressive enough, that person would receive a bounty for it. The bounty reward offered at the time was as follows: one song bundled with client (for a limited period of time), $50 osu!store credit, 6 months supporter tag, and a unique profile badge. This open bounty was reportedly created due to a want to reward players for quality beatmaps and to celebrate the new tracks but without having to go through the hassle that comes with running traditional mapping contests [[26]][r].

Reviewer 3: ```suggestion
In addition to this, an "open bounty" named "Mapping With Rewards" was created for this event, that allowed anyone to map one of the aforementioned tracks, get it into a rankable state, and if it was impressive enough, that person would receive a bounty for it. The bounty reward offered at the time was as follows: one song bundled with client (for a limited period of time), $50 osu!store credit, 6 months supporter tag, and a unique profile badge. This open bounty was reportedly created due to a need to reward players for quality beatmaps and to celebrate the new tracks without having to go through the hassle that comes with running traditional mapping contests [[26]][r].
```

Despite [promises from peppy on his blog](https://blog.ppy.sh/post/132009865043/mapping-with-rewards-oct-2015 "ppy blog"), however, the results for Mapping With Rewards were not posted by the end of the year.

<!--I don't think we should be mentioning anything about future years in the history posts as I want to make the posts as seamless as possible so that we only have to update the current (previous) year and not go back and fix anything, thus im not mentioning any future competitions, nor am I mentioning the results of the competition, as they were announced in 2016 (delete before merge)-->

## October

> After many months of work, a full migration from [DirectX](https://en.wikipedia.org/wiki/DirectX "Wikipedia") to [OpenGL](https://en.wikipedia.org/wiki/OpenGL "Wikipedia") graphics rendering was completed and pushed out to all users on the Stable release stream on 16 October 2015; however, users who were not able to run the new update due to compatibility issues were switched to a new release stream named "Stable (Fallback)", which used the old DirectX graphics rendering and was meant to be continuously supported alongside the Stable release until the team felt confident enough in the compatibility of the OpenGL rendering for all users [[28,29,30,31,32]][r]. <!--not sure if we should elaborate on this more or not (delete before merge)-->

Reviewer 3: ```suggestion
After months of work, a full migration from [DirectX](https://en.wikipedia.org/wiki/DirectX "Wikipedia") to [OpenGL](https://en.wikipedia.org/wiki/OpenGL "Wikipedia") graphics rendering was completed and pushed out to all users on the Stable release stream on 16 October 2015. However, users who were not able to run the new update due to compatibility issues were switched to a new release stream named "Stable (Fallback)", which used the old DirectX graphics rendering. It was meant to be continuously supported alongside the Stable release until the team felt confident enough in the compatibility of the OpenGL rendering for all users [[28,29,30,31,32]][r]. <!--not sure if we should elaborate on this more or not (delete before merge)-->
```

Another batch of official osu! songs were released by cYsmix on 27 October 2015, with a similar "open-bounty–style" contest being opened for it as well. The three songs were ["Fright March"](https://soundcloud.com/olemlanglie/cysmix-fright-march-osu "SoundCloud"), ["Moonlight Sonata"](https://soundcloud.com/olemlanglie/cysmix-moonlight-sonata-osu "SoundCloud"), and ["Classic Pursuit"](https://soundcloud.com/olemlanglie/cysmix-classic-pursuit-osu "SoundCloud") [[33,34]][r].

![](img/cysmix-album-cover-2.jpg "The album cover for cYsmix's second release of official osu! songs")

Just like the [previous month's Mapping with Rewards contest](#september), the results were not posted before the end of the year.

> The URL for the testing subdomain of the new osu! forums (jizz.ppy.sh) was replaced with new.ppy.sh on 26 October 2015 due to the sexual connotations that the word "jizz" in the old URL may have implied [[34,35]][r].

Reviewer 3: ```suggestion
The URL for the testing subdomain of the new osu! forums (jizz.ppy.sh) was replaced with new.ppy.sh on 26 October 2015 due to the explicit connotations that the old URL may have implied [[34,35]][r].
```

not sure if explicit is a correct word for this

> - All users which were targeted so far have used the same password for 3+ years.
> - Many used a very simple password, or shared the password across other services.
> - All users compromised have had at least one other account compromised in the past.
>
> Nonetheless, I am still taking further action:
>
> - I’ve expanded the list of common passwords, and am forcefully resetting passwords of users which have these.
> - We are on high alert and ready to help users out which are experiencing troubles.
> - We’ve made procedures for users which have invalid email addresses so they can recover their account successfully.
> - Working on adding two-factor support (email for now) to the osu! client. Should surface very soon.
>
> We have a lot of accounts to forcefully reset due to weak passwords, and due to limited support staff, will be staggering the “rollout” of these resets over the coming weeks.

— peppy, "20151119" [[40]][r]

<!--needs information regarding public opinion towards this change; section should aim to be as unbiased and neutral as possible-->

<!--the names of some of the streamers and their take on this would be appreciated here-->

> <!--resources regarding the xsplit breach: 

Reviewer: is this intentionally left here? also what's this neutral and unbiased crap? it's not like this was a breach on our end or anything?

Author: Huh, I thought I deleted that earlier (along with the rest of the "delete before merge" comments). I'll remove those.

Also, yeah, now that you mention it, that neutral and unbiased comment sounds a bit daft. I guess I just felt I that I was gaining so much information from your blog that I was worried from getting only one side of the story. But, yeah, that doesn't really need to be there
